met-api/migrations/versions/141eb11bdffc_merge_heads.py

```diff
@@ -0,0 +1,24 @@
+"""merge heads
+
+Revision ID: 141eb11bdffc
+Revises: e3427bc518f5, f99eb7f53041
+Create Date: 2023-03-27 15:02:24.649423
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '141eb11bdffc'
+down_revision = ('e3427bc518f5', 'f99eb7f53041')
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    pass
```

Comment:
empty migration shouldn't ever happen

Reply:
Yes theres different head files with the other prs